api/routes.go

```diff
@@ -1,8 +1,18 @@
 package api
 
 const (
-	// ProcessEndpoint is the endpoint for creating a new voting process
-	ProcessEndpoint = "/process"
 	// PingEndpoint is the endpoint for checking the API status
 	PingEndpoint = "/ping"
+	// ProcessesEndpoint is the endpoint for creating a new voting process
+	ProcessesEndpoint = "/processes"
+	// ProcessEndpoint is the endpoint to get the process info
+	ProcessURLParam = "processId"
+	ProcessEndpoint = "/processes/{" + ProcessURLParam + "}"
```

Comment:
I'd hardcode the full URL, I think it is OK and more readable. 

Consider making all these constants private.

Reply:
I changed it because in this way we can reference the param to get the data in its place ensuring that if it changes, the new value is propagated in every place.

Reply:
If we make them private, then they will not be accessible by other packages like `api/client`...